### PR TITLE
feat:rois add user get

### PR DESCRIPTION
**Pull request by roisalhaqq**

### CODE REVIEW

File: local_configuration/config.yaml
Comment (hnfnfl) [inline local_configuration/config.yaml:1]: do not commit this

File: internal/pkg/handler/handler.go [hunk 1]
```diff
@@ -41,6 +41,7 @@ func Run(cfg *configuration.Configuration, log *logrus.Logger) error {
 	nurseGroup := router.Group("/v1/user/")
 	nurseGroup.Use(middleware.JWTAuthMiddleware(cfg))
 	nurseGroup.POST("nurse/register", userHandler.Register)
+	nurseGroup.GET("", userHandler.GetUser)
 	nurseGroup.POST("nurse/{userId}/access", nurseHandler.AccessNurse)
```
Comment (hnfnfl) [inline internal/pkg/handler/handler.go]: please change this to `:userId` instead of `{userId}`

File: internal/pkg/handler/user.go [hunk 2]
```diff
@@ -117,3 +117,29 @@ func extractRole(path string) string {
 	}
 	return ""
 }
+
+func (h *UserHandler) GetUser(ctx *gin.Context) {
+	queryParams := ctx.Request.URL.Query()
+	var param dto.ReqParamUserGet
+
+	param.UserID = queryParams.Get("userId")
+	limit, _ := strconv.Atoi(queryParams.Get("limit"))
```
Comment (hnfnfl) [inline internal/pkg/handler/user.go]: add error handling

File: internal/pkg/handler/user.go [hunk 3]
```diff
@@ -117,3 +117,29 @@ func extractRole(path string) string {
...
+	param.Limit = limit
+	offset, _ := strconv.Atoi(queryParams.Get("offset"))
```
Comment (hnfnfl) [inline internal/pkg/handler/user.go]: add error handling

File: local_configuration/config.yaml
Comment (roisalhaqq) [reply local_configuration/config.yaml:1]: how to ignore this? we already add this folder in config yaml

**Pull request merged**